code-challenges.js

```diff
@@ -2,8 +2,9 @@
 
 
 // --------------------1) Create a function that returns the first 10 numbers of the Fibonacci sequence in an array. Expected output: [1, 1, 2, 3, 5, 8, 13, 21, 34, 55]
+const fibonacci = () => {
```

Comment:
there isnt a solution or pseudocode here. id love to know what your idea would be to solve this in pseudocode.